Foosball/Foosball/Tournament_bracket.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Foosball
+{
+    public partial class Tournament_bracket : Form
+    {
+        private static Random rng = new Random();
+        public Tournament_bracket(List<String> teamNames, int round = 1)
+        {
+            if(round == 1) {
+                var shuffledTeamNames = teamNames.OrderBy(a => Guid.NewGuid()).ToList();
+                teamNames = shuffledTeamNames;
+            }
+            checkIfEnded(teamNames);
+            InitializeComponent();
+            removeLabel(amountFinder(teamNames.Count));
+            addTeamNames(teamNames, amountFinder(teamNames.Count), round);
```

Comment:
amountFinder(teamNames.Count) could be refactored into variable, because you're repeating it two times